source/Preamble/04_requirements.rst

```diff
@@ -1,25 +1,24 @@
-Requirements (Informative)
-==========================
+Guiding Principles (Informative)
+===============================
 
-This document was developed to fulfill all requirements specified in **ADA Standard 1100 Functional Profiles for Orthodontic Photography** by the orthodontic community thus setting a standard methodology that clearly defines how orthodontic photographic images should be electronically represented:
+The OIP Profile was developed to fulfill the requirements specified in `ADA Standard 1100 Functional Profiles for Orthodontic Photography <https://www.ada.org/-/media/project/ada-organization/ada/ada-org/files/resources/practice/dental-standards/aip-review/1100_aip_review_july_2025.pdf?rev=480c3b96098849d39c713335ed951caf&hash=F1D0D7EDF2437B5F64D10252BF4DAB1D>`_ by the orthodontic community, thus setting a standard methodology that clearly defines how orthodontic photographic images should be electronically represented.  These principles guided OIP Profile development
 
 1. **Using DICOM**: 
    - DICOM provides the industry with the ability to develop solutions that store and recall images and their metadata across different products.
```

Comment:
I think we might want to add here that it the required format for dental images as defined by ADA-1114. (We could actually use that same wording.), which is really why we need to use it. The message being "See ADA-1114 for reasons why"